templates/service/build.sbt

```diff
@@ -19,6 +19,8 @@ lazy val microservice = Project(appName, file("."))
       "uk.gov.hmrc.hmrcfrontend.views.html.components._",
       "uk.gov.hmrc.hmrcfrontend.views.html.helpers._"
     ),
+    pipelineStages in Assets := Seq(gzip),
+    PlayKeys.devSettings ++= Seq("metrics.enabled" -> "false"),
```

Comment:
Normally bootstrap configures for development, and then values are overriden in app-configs. Doing this a different way may just catch people out?